Dockerfile

```diff
@@ -6,13 +6,14 @@ FROM tensorflow/tensorflow:1.10.1-gpu-py3
 ENV LC_ALL C.UTF-8
 ENV LANG C.UTF-8
 
-# Install misc. packages, GDAL, jq (for parsing extra-requirements.json), and Tippecanoe
+# Install misc. packages, GDAL, jq (for parsing extra-requirements.json), Tippecanoe dependencies, and OpenCV
 RUN add-apt-repository ppa:ubuntugis/ppa && \
     apt-get update && \
     apt-get install -y wget=1.* git=1:2.* python-protobuf=2.* python3-tk=3.* \
-            gdal-bin=2.1.* \
-            jq \
-            build-essential libsqlite3-dev zlib1g-dev && \
+                       gdal-bin=2.1.* \
+                       jq \
+                       build-essential libsqlite3-dev zlib1g-dev \
+                       libopencv-dev python-opencv && \
```

Comment:
Primarily because they were not pinned before: https://github.com/azavea/raster-vision/blob/ba83d57a1b1b65fe8b5693ed935f22804723fba3/Dockerfile#L27 https://github.com/azavea/raster-vision/blob/ba83d57a1b1b65fe8b5693ed935f22804723fba3/Dockerfile#L52-L54 .  Not a problem.